kubernetes/data_source_kubernetes_config_map.go

```diff
@@ -15,11 +15,13 @@ func dataSourceKubernetesConfigMap() *schema.Resource {
 			"metadata": namespacedMetadataSchema("config_map", false),
 			"data": {
 				Type:        schema.TypeMap,
+				Elem:        &schema.Schema{Type: schema.TypeString},
 				Description: "A map of the config map data.",
```

Comment:
To quote [tfproviderlint](https://github.com/bflad/tfproviderlint/tree/main/passes/S006):

```
The S006 analyzer reports cases of TypeMap schemas missing Elem, which currently passes Terraform schema validation, but breaks downstream tools and may be required in the future.
```

I tested this particular instance by creating a configmap resource using a released version of the provider and then doing a `terraform init -upgrade` to the test version of the provider (built from this branch). This schema changes doesn't seem to have any user-visible impact.